pokemonterminal/terminal/adapters/conemu.py

```diff
@@ -10,17 +10,16 @@ class ConEmuProvider(_TProv):
     def is_compatible() -> bool:
         return "CONEMUPID" in os.environ
 
+    def __run_command(command: str):
```

Comment:
A few places in this PR you have created "private" methods using name mangling with `__`. In general this isn't really recommended and using a single underscore to artificially denote methods & members as being "private" is preferred https://stackoverflow.com/a/7456912

Reply:
This is the style that was used in the code before (see the darwin adapter and database.py) so I just followed it for consistency.